src/pages/ServiceDetails/ServiceDetailsPage.tsx

```diff
@@ -182,14 +195,19 @@ class ServiceDetails extends React.Component<ServiceDetailsProps, ServiceDetails
   doRefresh = () => {
     const currentTab = this.state.currentTab;
 
-    if (currentTab === 'info') {
+    if (currentTab === defaultTab) {
       this.setState({ trafficData: null });
       this.fetchBackend();
     }
 
     if (currentTab === trafficTabName) {
       this.fetchTrafficData();
     }
+
+    if (currentTab === tracesTabName) {
+      this.fetchBackend();
```

Comment:
Do you need to fetch the backend when accessing to traces tab ?

Reply:
now not, I added this to update the number of errors traces when user go to traces. I'll remove it